### DISCUSSION TITLE
YAMLException:

### DISCUSSION THREAD

**Discussion by jiaicon** [hexojs/hexo#5072] [Question / Answer]
```
I got a wrong when I used 'hexo-generator-baidu-sitemap'.
I feel that there is a problem with the current version.
```

## Check List

Please check followings before submitting a new issue.

- [x] I have already read [Docs page](https://hexo.io/docs/) & [Troubleshooting page](https://hexo.io/docs/troubleshooting)
- [x] I have already searched existing issues and they are not help  to me
- [x] I examined error or warning messages and it's difficult to solve
- [x] Using [the latest](https://github.com/hexojs/hexo/releases) version of Hexo (run `hexo version` to check)
- [x] Node.js is higher than [minimum required version](https://hexo.io/docs/#Minimum-required-Node-js-version)

## Expected behavior
Normal clean with `hexo c`.
## Actual behavior
```
WARN  YAMLException: please see https://github.com/hexojs/hexo/issues/4917
FATAL
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
## How to reproduce?

* Step1
* Step2
* etc...

## Environment & Settings

**Node.js & npm version(`node -v && npm -v`)**

```
node: v16.14.0
npm: 8.3.1
```

**Your site `_config.yml`** (Optional)

```yaml
sitemap:
  path: sitemap.xml
baidusitemap:
  path: baidusitemap.xml
```

**Hexo and Plugin version(`npm ls --depth 0`)**

```
├── hexo-admin@2.3.0
├── hexo-baidu-url-submit@0.0.6
├── hexo-deployer-git@3.0.0
├── hexo-filter-github-emojis@3.0.5
├── hexo-generator-archive@1.0.0
├── hexo-generator-baidu-sitemap@0.1.9
├── hexo-generator-category@1.0.0
├── hexo-generator-feed@3.0.0
├── hexo-generator-index@2.0.0
├── hexo-generator-search@2.4.3
├── hexo-generator-sitemap@3.0.1
├── hexo-generator-tag@1.0.0
├── hexo-permalink-pinyin@1.1.0
├── hexo-renderer-ejs@2.0.0
├── hexo-renderer-marked@5.0.0
├── hexo-renderer-stylus@2.1.0
├── hexo-server@3.0.0
├── hexo-theme-landscape@0.0.3
├── hexo-wordcount@6.0.1
└── hexo@6.2.0
```

**Your package.json `package.json`**

```
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "scripts": {
    "build": "hexo generate",
    "clean": "hexo clean",
    "deploy": "hexo deploy",
    "server": "hexo server"
  },
  "hexo": {
    "version": "6.2.0"
  },
  "dependencies": {
    "hexo": "^6.2.0",
    "hexo-admin": "^2.3.0",
    "hexo-baidu-url-submit": "^0.0.6",
    "hexo-deployer-git": "^3.0.0",
    "hexo-filter-github-emojis": "^3.0.5",
    "hexo-generator-archive": "^1.0.0",
    "hexo-generator-baidu-sitemap": "^0.1.9",
    "hexo-generator-category": "^1.0.0",
    "hexo-generator-feed": "^3.0.0",
    "hexo-generator-index": "^2.0.0",
    "hexo-generator-search": "^2.4.3",
    "hexo-generator-sitemap": "^3.0.1",
    "hexo-generator-tag": "^1.0.0",
    "hexo-permalink-pinyin": "^1.1.0",
    "hexo-renderer-ejs": "^2.0.0",
    "hexo-renderer-marked": "^5.0.0",
    "hexo-renderer-stylus": "^2.1.0",
    "hexo-server": "^3.0.0",
    "hexo-theme-landscape": "^0.0.3",
    "hexo-wordcount": "^6.0.1"
  }
}
```

**Comment by yoshinorin** [comment]
@jiaicon 
Not sure but I assume this problem is out-of-hexo and its official-plugins. (we don't know unofficial plugins and themes)
Under the specific conditions a `YAMLException` warning message is occur as you see. See #4917 for more details.

```sh
WARN  YAMLException: please see https://github.com/hexojs/hexo/issues/4917
```

But, it is just `WARN`. As far as I know not to occur `FATAL`.  Not sure but I assume this problem is caused by `yarn` or related to it.
